src/ManojN/Q1ArrayMissingNumber.java

```diff
@@ -0,0 +1,23 @@
+//Q1) Find the missing number in integer array of 1 to 10?
+
+package a2;
+
+public class Q1ArrayMissingNumber {
+	void m1(int arr2 []){
+		// int arr2[]= {1,2,4,5,6};
+		int count=0;
+		for ( int i=0; i<arr2.length; i++){
+			
+			count=arr2[i]+1;
+			if ( count != arr2[i+1] ){
```

Comment:
Hi @ManojNAug19 
here this logic is same as [n*(n+1)]/2
you are adjusting the i's value... better to go with Mathematics logic this is also correct..